js/schema/section.json

```diff
@@ -0,0 +1,51 @@
+{
+  "type": "object",
```

Comment:
I would suggest something that is less general than `section`, maybe `formListSection`